Ruby/Ruby.sublime-syntax

```diff
@@ -68,7 +68,7 @@ file_extensions:
   - thor
   - Thorfile
   - Vagrantfile
-first_line_match: ^#!/.*\bruby
+first_line_match: ^#!\s*/.*\bruby\b
```

Comment:
I added the trailing `\b` here based on the Python regex, but then realized that all the first line regices are pretty different.  I'm happy to revert if this might break anything.